tracer/missing-nullability-files.csv

```diff
@@ -340,6 +340,7 @@ src/Datadog.Trace/Iast/Analyzers/UserStringInterop.cs
 src/Datadog.Trace/Iast/Aspects/DebugAspects.cs
 src/Datadog.Trace/Iast/Dataflow/AspectFilter.cs
 src/Datadog.Trace/Iast/Dataflow/AspectType.cs
+src/Datadog.Trace/LibDatadog/ServiceDiscovery/ResultTag.cs
```

Comment:
Can you add `#nullable enable` to this file? We never want to _add_ files to this list, only remove them.

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/7014/commits/89d1fc49b4dcf11799a838fd8ca079ac1846788d#diff-093e7ac26ece59c12cdd808c3bde66ca6fd1ae6937c8a270eb310037a05946f1